### ISSUE TITLE
Golang build tags

### ISSUE THREAD

**Issue by jkeys089** [GoogleCloudPlatform/buildpacks#247]
It seems there is no way to specify build tags for Go.

Would you be open to a PR to add this capability? If so, would `GOOGLE_GOTAGS` work for the env var name?

**Comment by matthewrobertson** [comment]
I haven't used Go build tags in the past so this isn't something I am very familiar with. But, I think this should already be supported with the `GOOGLE_GOGCFLAGS ` env var: https://cloud.devsite.corp.google.com/docs/buildpacks/go#compilation_flags

I think you would need to do something like:

```
pack build sample-go --builder=gcr.io/buildpacks/builder \
  --env GOOGLE_GOGCFLAGS="-tags=foobar"
```

**Comment by jkeys089** [comment]
@matthewrobertson thanks for your response. 

`GOOGLE_GOGCFLAGS` is used for `-gcflags` which is limited to the [go tool compile options](https://pkg.go.dev/cmd/compile).

However, I just saw in the docs that `GOFLAGS` (without a `GOOGLE_` prefix) is supported so I think that would work. I actually ended up switching to manual builds for better control / smaller images but hopefully this would help anyone interested in using buildpacks with build tags in the future.
